File: Projeto_2/Projeto_2/src/cliente.js
class Cliente {
  #cpf;

  constructor(nome, cpf) {
    this.nome = nome;
    this.#cpf = cpf;
  }

  getCpf() {
    return this.#cpf;
  }

  exibirInformacoes() {
    return `Nome: ${this.nome}, CPF: ${this.getCpf()}`;
  }
}

export default Cliente;

File: Projeto_2/Projeto_2-Ailton/src/cliente.py
class Cliente:
    def __init__(self, nome, cpf):
        # Inicializa os atributos do cliente
        self.nome = nome
        self.__cpf = cpf # encapsulamento do CPF do cliente

    def exibir_informacoes(self):
        # Exibe as informações do cliente, incluindo o CPF encapsulado
        print(f"Cliente: {self.nome} - CPF: {self.__cpf}")

    def get_cpf(self):
        # Método para acessar o CPF encapsulado
        return self.__cpf